test/endpoints_test.rb

```diff
@@ -0,0 +1,239 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class EndpointsTest < Minitest::Test
+  def setup
+    @stage_url = "https://stage.validators.app/api/v1"
+    @testnet_network = "testnet"
+    @mainnet_network = "mainnet"
+    @account = "DDnAqxJVFo2GVTujibHt5cjevHMSE9bo8HJaydHoshdp" # BlockLogic
+  end
+
+  def test_get_ping_returns_correct_json
+    expected_response = { "answer": "pong" }
```

Comment:
```suggestion
    expected_response = { "answer": "pong" }.to_json
```
I'd consider using `to_json` here and removing `transform_keys` below.